Formula/ghostscript.rb

```diff
@@ -1,17 +1,15 @@
 class Ghostscript < Formula
   desc "Interpreter for PostScript and PDF"
   homepage "https://www.ghostscript.com/"
-  url "https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs9561/ghostpdl-9.56.1.tar.xz"
-  sha256 "05e64c19853e475290fd608a415289dc21892c4d08ee9086138284b6addcb299"
+  url "https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs1000/ghostpdl-10.0.0.tar.xz"
+  sha256 "8f2b7941f60df694b4f5c029b739007f7c4e0d43858471ae481e319a967d5d8b"
   license "AGPL-3.0-or-later"
-  revision 1
 
   # We check the tags from the `head` repository because the GitHub tags are
   # formatted ambiguously, like `gs9533` (corresponding to version 9.53.3).
   livecheck do
-    url :stable
-    regex(/href=.*?ghostpdl[._-]v?(\d+(?:\.\d+)+)\.t/i)
-    strategy :github_latest
+    url :head
+    regex(/^ghostpdl[._-]v?(\d+(?:\.\d+)+)$/i)
```

Comment:
```suggestion
    url "https://www.ghostscript.com/json/settings.json"
    regex(/["']GS_VER["']:\s*?["']v?(\d+(?:\.\d+)+)["']/i)
```

Checking the Git tags technically works but the release tarball may not be available yet when the tag is created (i.e., this is the reason behind using the `GithubLatest` strategy here). The [first-party download page](https://www.ghostscript.com/releases/gpdldnld.html) links to the latest GhostPDL tarball on GitHub but the link is generated using JavaScript from a JSON file. The suggestion above matches the version from the JSON file, so we shouldn't have issues with livecheck surfacing a new version before the tarball is available.

In the future, I'll update this to use a `Json` strategy (which will properly parse the JSON and extract the value for the `GS_VER` field) but I still have to work my way to creating that PR (i.e., the work's done but I have some other brew PRs to tackle first).